.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+## Description
+
+A short summary of the idea.
+
+## How will this impact [SecureDrop users](https://github.com/freedomofpress/securedrop-ux/wiki/Users)?
+<!-- How do you feel this change might impact SecureDrop's usability, accessibility, or usefulness—and specifically, for which users? Has anecdotal feedback from users influenced this change? Does evidence exist from user research to support this idea? Could design or user research efforts be helpful to support a change? -->
+
+## How would this affect SecureDrop's [threat model](https://docs.securedrop.org/en/stable/threat_model/threat_model.html)?
+<!-- Would this change create new risks for sources, journalists, or administrators? Would it mitigate existing risks? -->
+
+## User Stories
+<!-- “As a [role], I want to [task], so that [reason].” -->
+
+If appropriate
```

Comment:
Was this accidentally left in, or is this how you want it to read? You could change it to "Any relevant user stories."